src/lib/db/group-store.ts

```diff
@@ -190,25 +209,31 @@ export default class GroupStore implements IGroupStore {
 
     async addUsersToGroup(
         groupId: number,
-        users: IGroupUserModel[],
+        users: ICreateGroupUserModel[],
         userName: string,
-        transaction?: Transaction,
     ): Promise<void> {
-        const rows = (users || []).map((user) => {
-            return {
-                group_id: groupId,
-                user_id: user.user.id,
-                created_by: userName,
-            };
-        });
-        return (transaction || this.db).batchInsert(T.GROUP_USER, rows);
```

Comment:
removing transactions from the store and moving them to the controller/around the group service in the enterprise repo